typhos/utils.py

```diff
@@ -166,9 +166,24 @@ def random_color():
 
 
 class TyphosLoading(QtWidgets.QLabel):
+    LOADING_TIMEOUT_MS = 10000
     loading_gif = None
     """Simple widget that displays a loading GIF"""
-    def __init__(self, *args, **kwargs):
+    def __init__(self, enable_timeout=True, *args, **kwargs):
+        """
+        A QLabel with an animation for loading status.
+
+        Attributes
+        ----------
+        LOADING_TIMEOUT_MS : int
+            The timeout value in milliseconds for when to stop the animation
+            and replace it with a default timeout message.
+
+        Parameters
+        ----------
+        enable_timeout : bool
```

Comment:
Maybe unnecessary; could leave it based on `if LOADING_TIMEOUT_MS > 0`

Reply:
Addressed via https://github.com/pcdshub/typhos/pull/331/commits/68ae4b6e9b26f8b11ad40f47aca6c8953d82bcd3